### PR TITLE
Add ticket view and assign ticket to developer

### PR DESCRIPTION
**Pull request by krzjez**

### CODE REVIEW

File: src/main/java/com/metabug/web/controller/IndexController.java [hunk 1]
```diff
@@ -4,13 +4,19 @@
...
 import org.springframework.web.servlet.ModelAndView;
 
+import java.security.Principal;
+
 @Controller
 public class IndexController {
 
-    @GetMapping(value = {"/login"})
-    public ModelAndView index() {
+    @GetMapping(value = {"/login", ""})
```
Comment (kujawad) [inline src/main/java/com/metabug/web/controller/IndexController.java:12]: Why `""`?

**Review by kujawad — APPROVED** [review]

File: src/main/java/com/metabug/web/controller/IndexController.java (hunk 1 above)
Comment (krzjez) [reply src/main/java/com/metabug/web/controller/IndexController.java:12]: Login page is on both /login and "", so if "" is not present, a logged user could access the login page.

Comment (kujawad) [reply src/main/java/com/metabug/web/controller/IndexController.java:12]: Login page is only on `/login`. Spring Security automatically creates filter chain to    redirect `/login`, if you want `""` to be accessible as login page, in my understanding you have to add `.loginPage("")` in configuration 

WebSecurityConfiguration:
```java
...
                .formLogin()
           ---> .loginPage("/login")            
                .failureUrl("/login?error=true")
                .defaultSuccessUrl("/home")
                .usernameParameter("login")
                .passwordParameter("password")
...
```

Comment (krzjez) [reply src/main/java/com/metabug/web/controller/IndexController.java:12]: Then explain this. It's only accessible for a logged user though.
image

Comment (kujawad) [reply src/main/java/com/metabug/web/controller/IndexController.java:12]: Got it, resolved.

**Review by kujawad — APPROVED** [review]

**Pull request merged**